albumentations/augmentations/transforms.py

```diff
@@ -446,6 +447,49 @@ def get_transform_init_args_names(self):
         return ("height", "width", "interpolation")
 
 
+class Scale(DualTransform):
+    """Scale the input to the given scaling factor.
+
+    Args:
+        scale (float or (float, float)): scaling factor. If scale is a single float value -
+            input will be scaled proportionately. If tuple of floats - different scaling factors
+            will be used for the width and height of the input.
+        interpolation (OpenCV flag): flag that is used to specify the interpolation algorithm. Should be one of:
+            cv2.INTER_NEAREST, cv2.INTER_LINEAR, cv2.INTER_CUBIC, cv2.INTER_AREA, cv2.INTER_LANCZOS4.
+            Default: cv2.INTER_LINEAR.
+        p (float): probability of applying the transform. Default: 1.
+
+    Targets:
+        image, mask, bboxes, keypoints
+
+    Image types:
+        uint8, float32
+    """
+
+    def __init__(self, scale, interpolation=cv2.INTER_LINEAR, always_apply=False, p=1):
+        super(Scale, self).__init__(always_apply, p)
+
+        if isinstance(scale, (int, float)):
```

Comment:
I believe this check is not going to make what you wanted to express.
In your case condition should sound like "If scale is tuple of integers or floats with two elements". In code, however it  checks whether scale is either `int` or `float`. 
The correct check would be

```
if isinstance(scale, (int, float)):
   self.scale = scale
elif isinstance(scale, (tuple,list)) and len(scale) == 2:
   self.scale = scale, scale
else:
   raise ValueError("Parameter scale must be either scalar or tuple of (scale_x, scale_y)")
```

Also, please add a unit test for this augmentation that verify that per-axis scale parameters produce expected output image dimensions.